docs/Build.md

```diff
@@ -137,6 +82,36 @@ These options will all be provided to the TerserPlugin via it's constructor.
 | terserOptions.keep_fnames     | `true`                                  |                   Keep original function names |
 | terserOptions.mangle.safari10 | `true`                                  |               Works around known Safari issues |
 
+##### Swappable view layers
+
+**TODO - this approach needs to change with use of NormalModuleReplacementPlugin**
```

Comment:
Would you mind raising an issue for this? Out of scope for this PR, but should be done sooner rather than later

Reply:
I believe it's in @dlabaj PR for typescript?

Reply:
yes it's in typescript pr

Reply:
Question for you both @dlabaj / @nictownsend - I have just been looking at #29 again. I can see the section that talks about file name converntion, but cannot see any mention nor integration of `NormalModuleReplacementPlugin` to enable the swapping of view layer used. What is the plan here? As a starter, I would suggest getting #29 in first, given its age/it is more doc/configuration based, where as this adds implementation of i18n as well.

Reply:
Agree - it was their prior to a refactor- will talk with @dlabaj to see what's ocurring